sktime/forecasting/tests/test_varmax.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+"""Tests the VARMAX model."""
+__author__ = ["KatieBuc"]
+
+import numpy as np
+import pandas as pd
+
+from numpy.testing import assert_allclose
+from sktime.forecasting.model_selection import temporal_train_test_split
+from statsmodels.tsa.api import VARMAX as _VARMAX
+from sktime.forecasting.base import ForecastingHorizon
+from sktime.forecasting.varmax import VARMAX
+
+index = pd.date_range(start="2020-01", end="2021-12", freq="M")
+df = pd.DataFrame(
+    np.random.randint(0, 100, size=(23, 3)),
+    columns=list("ABC"),
+    index=pd.PeriodIndex(index),
+)
```

Comment:
if we are creating a random dataset in testing, would be good to ensure this is reproducible. E.g. setting random seed, or creating rng object in numpy. (I unfortunatley cannot remember best way to do this)

Reply:
yeah, true, okay I set a seed